packages/genesys-spark-tokens/snapshot-new/scss/gse-legacy-light-semantic.scss

```diff
@@ -14,6 +14,7 @@
   --gse-semantic-opacity-shroud: 0.8;
   --gse-semantic-theme-fontFamily-headings: Roboto;
   --gse-semantic-theme-fontFamily-body: Roboto;
+  --gse-semantic-theme-fontFamily-monospace-body: 'Noto Sans Mono';
```

Comment:
If this is for the code examples I would call the token something like:
`--gse-semantic-theme-fontFamily-code-body` or `--gse-semantic-theme-code-FontFamily`

The structure of the token name may need to be looked at.
https://genesys-confluence.atlassian.net/wiki/spaces/PureCloud/pages/324841653/COMUI-1733+Design+token+naming+conventions